js/Track/Model.js

```diff
@@ -111,9 +120,29 @@ module.exports = Base.extend({
     }
 
     $.when.apply($, $.map(bins, function (bin) {
+      
+      /* DEBUG FOR SAP-25391 */
+      var debug = {}
+      
+      if ([chr, bin[0], bin[1]].includes(NaN)) {
+        debug = {
+          start: start,
+          end: end,
+          length: length,
+          initData: initData,
+          dataRequestLimit: this.dataRequestLimit,
+          allData: this.allData,
+          userAgent: navigator.userAgent,
+        };
+      }
+      /* END OF DEBUG */
+      
       var request = $.ajax({
         url       : model.parseURL(chr, bin[0], bin[1]),
-        data      : model.urlParams,
+        data      : {
+          debug,
```

Comment:
Is BE expect additional fields here?

Also I think it worth adding a `debug` comment here as well.